tree/dataframe/inc/ROOT/RDF/RDataBlockID.hxx

```diff
@@ -25,9 +26,13 @@ namespace RDF {
 class RDataBlockID {
    // Currently backed by a simple string, might change in the future as we get usage experience.
```

Comment:
Obsolete after entry range was introduced?

Reply:
could be removed, yes